eg/Classes/PluginItem.py

```diff
@@ -78,6 +78,16 @@ def SearchFunc(obj):
             return False
         if not TreeItem.AskDelete(self):
             return False
+
+        if self.info.persistentData.config:
+            answer = eg.MessageBox(
+                eg.text.General.deleteConfigMessage,
```

Comment:
I think in my original version i did put in plugin configuration data... but somewhere along the line.. it got misplaced? LOL